pkg/autoops/api/progressive_rollout_operation.go

```diff
@@ -138,21 +138,17 @@ func updateFeatureTargeting(
 	feature *featureproto.Feature,
 	targetVariationID, environmentNamespace string,
 ) error {
-	cmds, err := getResetFeatureCmds(feature)
+	c, err := getChangeDefaultStrategyCmd(feature, schedule, targetVariationID)
 	if err != nil {
 		return err
 	}
-	c, err := getNewRuleCmd(feature, schedule, targetVariationID)
-	if err != nil {
-		return err
-	}
-	cmds = append(cmds, c)
 	_, err = featureClient.UpdateFeatureTargeting(
 		ctx,
 		&featureproto.UpdateFeatureTargetingRequest{
 			EnvironmentNamespace: environmentNamespace,
 			Id:                   feature.Id,
-			Commands:             cmds,
+			Commands:             []*featureproto.Command{c},
+			From:                 featureproto.UpdateFeatureTargetingRequest_OPS,
```

Comment:
All requests from the progressive rollout must have set the `from` field as `OPS`.